WebAndLoadTests/README.md

```diff
@@ -1,7 +1,28 @@
 # Web and Load Test
 
-## Edit app.config 
+## Prerequisites
+1. Visual Studio
+	- [Download and install Visual Studio](https://www.visualstudio.com/downloads/)
+2. VSTS Account
+
+## Setting Up Config Values in app.config 
 - Input the value of the base URL for the app in Line 11 of ```app.config```. Example: ```<value>URL</value>```.
 - Input the value of the base URL for the middle tier API in Line 14 of ```app.config```. Example: ```<value>URL</value>```.
 - Input the value of the username of the admin user that has the access to delete users in Line 17 of ```app.config```. Example: ```<value>USERNAME</value>```.
-- Input the value of the password of the admin user that has the access to delete users in Line 20 of ```app.config```. Example: ```<value>PASSWORD</value>```.
\ No newline at end of file
+- Input the value of the password of the admin user that has the access to delete users in Line 20 of ```app.config```. Example: ```<value>PASSWORD</value>```.
+
+## Web Test
+### Running and Debugging Web Test
+- Right click on the WebTest class and select ```Run Coded Web Performance Test``` to run your web test or select ```Debug Coded Web Performance Test```  to run your web test.
+
+## Load Test on Visual Studios
```

Comment:
I would change this to "Executing the Load Test"; remove the specific reference to Visual Studio since this can be executed in VSTS or in Visual Studio.